pokemongo_bot/api_wrapper.py

```diff
@@ -8,13 +8,18 @@
 from pgoapi.pgoapi import PGoApi, PGoApiRequest, RpcApi
 from pgoapi.protos.POGOProtos.Networking.Requests_pb2 import RequestType
 
-from human_behaviour import sleep
+from human_behaviour import sleep, gps_noise_rng
 
 class ApiWrapper(PGoApi):
-    def __init__(self):
+    def __init__(self, replicate_gps_noise=False, gps_noise_range=0.00025):
         PGoApi.__init__(self)
         self.useVanillaRequest = False
 
+        if replicate_gps_noise:
+            self.gps_noise_range = gps_noise_range
```

Comment:
The default is 8 for altitude according to the API, and this does not change. Changing the altitude by a random factor is required, however like @kanemasa1987 said, this should be separate from the gps position (2 configuration parameters, the default height and the maximum variation (default 15))